server/src/core/server/app/router/api/dsaReport.ts

```diff
@@ -0,0 +1,15 @@
+import { AppOptions } from "coral-server/app";
+import { reportDownloadHandler } from "coral-server/app/handlers";
+import { userLimiterMiddleware } from "coral-server/app/middleware/userLimiter";
+
+import { createAPIRouter } from "./helpers";
+
+export function createDSAReportRouter(app: AppOptions) {
+  const router = createAPIRouter({ cacheDuration: "30s" });
+
+  router.use(userLimiterMiddleware(app));
```

Comment:
I would recommend also putting an ip limiter on this

Reply:
Good point, updating to do that, thank you!